e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -229,13 +225,56 @@ describe("scenarios > embedding > dashboard parameters", () => {
 
       cy.log("should accept url parameters");
 
-      cy.url().then(url => cy.visit(url + "?id=1&id=3"));
+      cy.location().then(location =>
+        cy.visit(`${location.origin}${location.pathname}?id=1&id=3`),
+      );
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
       cy.contains(".ScalarValue", "2");
     });
   });
 });
 
+describe("scenarios > embedding > dashboard parameters with defaults", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createNativeQuestionAndDashboard({
+      questionDetails: questionDetailsWithDefaults,
+      dashboardDetails,
+    }).then(({ body: { id, card_id, dashboard_id } }) => {
+      cy.wrap(dashboard_id).as("dashboardId");
+
+      mapParameters({ id, card_id, dashboard_id });
+    });
+
+    cy.get("@dashboardId").then(dashboardId => {
+      visitDashboard(dashboardId);
+    });
+  });
+
+  it("card parameter defaults should apply for disabled parameters, but not for editable or locked parameters", () => {
+    cy.icon("share").click();
+    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
```

Comment:
any way to get rid of eslint-disable?

Reply:
Ah yes, I forgot to clean these up. I will do this tomorrow